packages/oscal-react-library/src/components/OSCALProperties.tsx

```diff
@@ -0,0 +1,146 @@
+import Dialog from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import Stack from "@mui/material/Stack";
+import { styled } from "@mui/material/styles";
+import React from "react";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+import StyledTooltip from "./OSCALStyledTooltip";
+import {
+  OSCALSystemImplementationTableTitle,
+  StyledHeaderTableCell,
+  StyledTableRow,
+  StyledTableHead,
+} from "./OSCALSystemImplementationTableStyles";
+
+const OSCALPropertiesButton = styled(Button)(
+  ({ theme }) => `
+    color: ${theme.palette.primary.main};
+    margin-top: 1em;
+    margin-bottom: 0.5em;
+  `
+);
+
+function OSCALPropertiesCard(props: any) {
+  const { title, children } = props;
+
+  const [open, setOpen] = React.useState(false);
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <OSCALPropertiesButton
+        size="small"
+        variant="outlined"
+        onClick={handleOpen}
+        aria-label={`${title} properties button`}
+        startIcon={<HomeIcon />}
+      >
+        Properties
+      </OSCALPropertiesButton>
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        scroll="paper"
+        aria-labelledby="scroll-dialog-title"
+        aria-describedby="scroll-dialog-description"
+        maxWidth="md"
+        fullWidth
+      >
+        {children}
+      </Dialog>
+    </>
+  );
+}
+
+function isNistNamespace(namespace: string) {
+  return !namespace || namespace.includes("nist.gov");
+}
+
+function getThirdPartyNamespaces(properties: any) {
+  let items: string[] = [""];
+
+  properties
+    ?.filter((property: any) => !isNistNamespace(property?.ns))
+    .map((property: any) => {
+      items.push(property?.ns);
+    });
+
+  return items;
+}
+
+function OSCALPropertiesTable(props: any) {
+  const { properties, namespace } = props;
+
+  return (
+    <DialogContent dividers>
+      <OSCALSystemImplementationTableTitle variant="h6" id={`${namespace}-table-title`}>
+        {namespace}
+      </OSCALSystemImplementationTableTitle>
+      <TableContainer sx={{ maxHeight: "25em" }}>
+        <Table aria-label="Components" sx={{ height: "max-content" }}>
+          <StyledTableHead>
+            <TableRow>
+              <StyledHeaderTableCell>Name</StyledHeaderTableCell>
+              <StyledHeaderTableCell>Class</StyledHeaderTableCell>
+              <StyledHeaderTableCell>Value</StyledHeaderTableCell>
+            </TableRow>
+          </StyledTableHead>
+          <TableBody>
+            {properties
+              ?.filter((property: any) =>
+                namespace === "NIST OSCAL"
```

Comment:
Make this a constant if we're going to reuse it -- especially bc i don't love this value but I can't think of something better right now.

Reply:
I agree, my plan was to check-in on what our NIST namespace should be called.